torch/distributed/checkpoint/utils.py

```diff
@@ -371,3 +374,22 @@ def _normalize_device_info(device_type: str, device_id: int) -> str:
     if device_type == "cpu":
         return "cpu"
     return f"{device_type}:{device_id}"
+
+
+# TODO: integrate with distributed logging flag
+ENABLE_PROFILE = False
```

Comment:
Just to make sure I understand correctly. The default is without the profiling unless manually change `ENABLE_PROFILE` to True. Is that correct?

Reply:
Yes, we don't want to enable this by default. We can integrate it with torch distributed logging later.